CHANGELOG.md

```diff
@@ -1,13 +1,11 @@
 
 ## v2.2.2
-### Unreleased
```

Comment:
Sorry, you should leave this one in.  But are you going to release it?  If so this should have a date 

```
### 2024-07-16
```

This is probably worth releasing, but I'm not sure who makes that call.

Reply:
Not sure ...